modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryRelatedObjectsResourceImpl.java

```diff
@@ -187,7 +194,10 @@ private void _checkRelatedObjectEntry(
 			String objectRelationshipName, long relatedObjectEntryId)
 		throws Exception {
 
-		objectEntryManager.getObjectEntry(
+		DefaultObjectEntryManager defaultObjectEntryManager =
```

Comment:
Maybe you can call the provider on method deleteCurrentObjectEntry and then pass DefaultObjectEntryManager to _checkCurrentObjectEntry and _checkRelatedObjectEntry

Reply:
Done [here](https://github.com/liferay-objects/liferay-portal/pull/2092/commits/d919100c400b135204ac34c85fcfb44559e1e97e)

Thanks!